test/src/test/java/org/corfudb/runtime/view/StreamViewTest.java

```diff
@@ -464,4 +468,79 @@ public void testPreviousWithStreamCheckpoint() throws Exception {
         assertThatThrownBy(() -> sv.previous()).isInstanceOf(TrimmedException.class);
         assertThat(sv.getCurrentGlobalPosition()).isEqualTo(baseVersion.getSequence());
     }
+
+    /**
+     * Test a transaction no longer need to call sequencer and follow backpointer
+     * if the accessed stream is already resolved up to the snapshot timestamp.
+     */
+    @Test
+    public void testStreamSyncNoExtraSequencerCall() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651479)